core/src/main/java/io/aiven/klaw/service/CacheService.java

```diff
@@ -0,0 +1,158 @@
+package io.aiven.klaw.service;
+
+import java.util.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.*;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Slf4j
+public class CacheService<T> {
```

Comment:
This class does not have a 'service' annotation. do we want to move this to util folder?